scripts/services/migrations.ts

```diff
@@ -1,4 +1,25 @@
 import { compat, types as T } from "../deps.ts";
 
 export const migration: T.ExpectedExports.migration = compat.migrations
-    .fromMapping( { }, "0.9.7.1" );
+.fromMapping({
+    "0.9.7.1": {
+        up: compat.migrations.updateConfig(
+            (config) => {
+                const implementation = config.wallet.type === 'LndRestWallet' ? 'lnd' : 'cln'
```

Comment:
this has a bug, wallet.type is not recognized

Reply:
I assume you are upgrading from a prior version where the config was never set/saved?

Reply:
no, I cannot build the s9pk: 
```

error: TS2571 [ERROR]: Object is of type 'unknown'.
                const implementation = config.wallet.type === 'LndRestWallet' ? 'lnd' : 'cln'
                                       ~~~~~~~~~~~~~
    at file:///Users/dread/Documents/Start9/Repos/lnbits-wrapper/scripts/services/migrations.ts:8:40
make: *** [scripts/embassy.js] Error 1
```

Reply:
👍